src/Resolvers/Microsoft.DotNet.WorkloadManifestReader/WorkloadManifest.cs

```diff
@@ -21,20 +21,30 @@ public static WorkloadManifest LoadFromFolder(string manifestFolder)
             {
                 var manifestXml = XDocument.Load(manifestPath);
 
-                foreach (var workload in manifestXml.Root.Element("Workloads").Elements("Workload"))
+                if (manifestXml.Root?.Element("Workloads")?.Elements("Workload") != null)
                 {
-                    string workloadName = workload.Attribute("Name").Value;
-                    var workloadPacks = workload.Elements("RequiredPack").Select(rp => rp.Attribute("Name").Value).ToList();
+                    foreach (var workload in manifestXml.Root.Element("Workloads").Elements("Workload"))
+                    {
+                        string workloadName = workload?.Attribute("Name").Value;
```

Comment:
minor: inconsistent use of ```var``` vs. explicit type declarations.

Reply:
we just rely on editorconfig. if it does not compliant, it is good